packages/ripple-address-codec/test/utils.test.ts

```diff
@@ -9,20 +9,20 @@ it('elements must be in the same order', () => {
 })
 
 it('sequences do not need to be the same type', () => {
```

Comment:
This test name should change to something like `Uint8Array is equivalent to other representations`

Reply:
Or `sequences with multiple values` to match the naming convention of the other `it(...)` cases

Reply:
Bump :)

Reply:
Addressed in 3568f04a61d7f3ea188e12d30b9c3e0497dbd0ed